src/main/java/phrasecount/pojos/PhraseAndCounts.java

```diff
@@ -0,0 +1,24 @@
+package phrasecount.pojos;
+
+public class PhraseAndCounts extends Counts {
+  public String phrase;
+
+  public PhraseAndCounts(String phrase, int docCount, int sum) {
```

Comment:
I made the var names more descriptive in 34c285e.   An example should not have obscure var names.